docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+  from `crypto-stellar.crypto_stellar.history_assets`
+  where true
+    -- equivalent asset_code/asset_issuer parameters
+    and asset_code=<asset code>
+    and asset_issuer=<asset issuer>
+),
+accounts as (
+  select
+    count(ac.account_id) as num_accounts
+    , countif(ac.flags=1) as authorized_accounts
+    , sum(balance) as amount
+  from assets
+    join `crypto-stellar.crypto_stellar_dbt.trust_lines_current` as ac
+      on assets.asset_code = ac.asset_code
+      and assets.asset_issuer = ac.asset_issuer
+  where true
+),
+liquidity_pools as (
+  select
+    count(*) as num_liquidity_pools
+    , sum(lpc.asset_a_amount) as liquidity_pools_amount
+  from assets
+    join `crypto-stellar.crypto_stellar_dbt.liquidity_pools_current` as lpc
+      on assets.asset_code = lpc.asset_a_code
+      and assets.asset_issuer = lpc.asset_a_issuer
+  where true
+),
+current_claimable_balances as (
+  -- TODO: This should be replaced with the claimable_balances_current table when available
+  SELECT
+    cb.balance_id
+    , cb.asset_code
+    , cb.asset_issuer
+    , cb.asset_amount
+    , cb.deleted
+    , cb.last_modified_ledger
+    , row_number() over(partition by cb.balance_id, cb.asset_code, cb.asset_issuer order by last_modified_ledger desc, deleted desc) as rn
+  FROM assets
+    join `crypto-stellar.crypto_stellar.claimable_balances` as cb
+      on assets.asset_code = cb.asset_code
+      and assets.asset_issuer = cb.asset_issuer
+  where true
+  order by last_modified_ledger asc
+),
+claimable_balances as (
+select
+    count(*) as num_claimable_balances
+    , sum(asset_amount) as claimable_balances_amount
+  from current_claimable_balances
+  where true
+    and rn = 1
+    and deleted = false
+)
+
+select
+  *
+from assets
+  join accounts on true
+  join liquidity_pools on true
+  join claimable_balances on true
+```
+
+### Getting Stellar Asset Contract Information for an Asset
+
+```sql
+select
+  count(balance_holder) as num_contracts
+  , sum(parse_bignumeric(balance)) * .0000001 as contracts_amount
+from `crypto-stellar.crypto_stellar_dbt.contract_data_current`
+where true
+  and contract_id = <SAC contract id>
+  and balance != ""
+```
+
+## Claimable Balances
+
+[horizon/claimable_balances](/docs/data/horizon/api-reference/resources/claimablebalances)
+
+A Claimable Balance represents the transfer of ownership of some amount of an asset. Claimable balances provide a mechanism for setting up a payment which can be claimed in the future. This allows you to make payments to accounts which are currently not able to accept them.
+
+### List All Claimable Balances
+
+```sql
+with assets as (
+  -- Used to filter to a specific asset
+  select
+    asset_code
+    , asset_issuer
+  from `crypto-stellar.crypto_stellar.history_assets`
+  where true
+    -- equivalent asset_code/asset_issuer parameters
+    and asset_code=<asset code>
+    and asset_issuer=<asset issuer>
+),
```

Comment:
nit: is this CTE actually needed? or can we pull asset info from claimable_balances itself